src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -92,6 +93,15 @@
 
 HEXDIGITS_SET = set(string.hexdigits)
 
+Strategies = collections.namedtuple('Strategies', [
```

Comment:
nit: this name seemed quite confusing to me, until I read the function below. Maybe something a bit more generic than `Strategies`, like `Parameters` (don't like it) or `StrategiesAndParameters` or some sort of `Config` or `Configuration`, just because "strategy" is used quite a lot referring to the actual strategies?

Reply:
+1 or StrategyMetadata or something

Reply:
Renamed to StrategyInfo.